src/rohini_deshmane/ConsonantFromString.java

```diff
@@ -0,0 +1,29 @@
+package rohini_deshmane;
+
+public class ConsonantFromString {
+/*
+ * Print last consonant from the string
+
+input : aakansha
+output : h
+ */
+	void printLastConsonant(String str) {
+		char consonant='-', ch;
+		for(int i=0; i< str.length(); i++) {
+			ch = str.charAt(i);
+			if(ch!='a' || ch!='A' || ch != 'e' || ch != 'E' || ch !='I' || 
```

Comment:
You can convert string to either lower or uppercase so that you handle in less conditions